### PR TITLE
Create a ref doc for pod security policy

### PR DESCRIPTION
**Pull request by pweil-**
@devin-donnelly 1.4 version as requested in https://github.com/kubernetes/kubernetes.github.io/pull/1150#issuecomment-246049730.  Updates have been made in both PRs.  Thanks!

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1206)

### CODE REVIEW

**Comment by devin-donnelly** [comment]
Docs LGTM. This does look like it needs a tech review.

File: docs/user-guide/pod-security-policy/index.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+1. persistentVolumeClaim
+1. rbd
+1. cinder
+1. cephFS
+1. downwardAPI
+1. fc
+1. configMap
+1. \* (allow all volumes)
+
+The recommended minimum set of allowed volumes for new SCCs are
```
Comment (erictune) [inline docs/user-guide/pod-security-policy/index.md]: s/SCC/PSP/

Comment (devin-donnelly) [inline docs/user-guide/pod-security-policy/index.md]: Should this be fixed before merge? I'll mark it as Tech Review: Open Issues.

Comment (pweil-) [inline docs/user-guide/pod-security-policy/index.md]: Yes.  This is updated now..

**Comment by devin-donnelly** [comment]
Thanks, merging now.

**Comment by nikhiljindal** [comment]
This PR seems to have broken deploy/netlify on release-1.4 branch.

It is failing on all PRs with the following error (Ref https://github.com/kubernetes/kubernetes.github.io/pull/1222#issuecomment-248142057):

```
3:27:46 PM: Liquid Exception: Could not locate the included file 'sj.yaml' in any of ["/opt/build/repo/docs/user-guide/pod-security-policy"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in docs/user-guide/pod-security-policy/index.md
3:27:46 PM: jekyll 3.2.1 | Error: Could not locate the included file 'sj.yaml' in any of ["/opt/build/repo/docs/user-guide/pod-security-policy"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source.
3:27:46 PM: make: *** [build-preview] Error 1
```

`docs/user-guide/pod-security-policy/index.md` added in this PR is referring to `sj.yaml` file that does not exist.

**Comment by pweil-** [comment]
looking

**Pull request merged**